src/tests/components/SignUp/SignUserAgreement.test.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  fireEvent, render, screen, waitFor,
+} from '@testing-library/react';
+import React from 'react';
+import { Provider, transitions } from 'react-alert';
+import AlertTemplate from 'react-alert-template-basic';
+import { MemoryRouter } from 'react-router-dom';
+
+import { SignUserAgreement } from '../../../components/SignUp/pages/SignUserAgreement';
+
+const options = {
+  position: 'bottom left',
+  timeout: 5000,
```

Comment:
do we need this?

Reply:
i think so, i talked w nick about this. Essentially, a lot of these tests fail when we try to render the component itself without the provider tag, so our work around was that we render the provider tag as well and these are the props you need for it.

Reply:
yeah i just took a look at this and i guess we only need the provider for the CompleteSignupflow test and not the other components, good catch